### PR TITLE
Debug Static Option.

### PR DESCRIPTION
**Pull request by KarinG**
Overview
----------------------------------------
Working on the switching Live -> Static Option issue

### CODE REVIEW

**Comment by KarinG** [comment]
Not sure why we get this in a test -> but it is working in the UI. 

image

**Comment by KarinG** [comment]
on my local instance [older versions] - I always have a form_key -> this is in the form yml right after hitting save on the CiviCRM Settings screen - so immediately after the Element has been created:

image

File: src/Plugin/WebformElement/CivicrmOptions.php [hunk 1]
```diff
@@ -215,4 +215,21 @@ public static function ajaxCallback(array $form, FormStateInterface $form_state)
...
+   * {@inheritdoc}
+   */
+  public function hasMultipleValues(array $element) {
+    if (!empty($element['#extra']['multiple']) || (!empty($element['#options']) && count($element['#options']) === 1)) {
+      return TRUE;
+    }
+    return FALSE;
+  }
+
 }
```
Comment (KarinG) [inline src/Plugin/WebformElement/CivicrmOptions.php]: I've lifted this out of this PR - to solve this Notice separately: 

```
  /**
   * {@inheritdoc}
   */
  public function hasMultipleWrapper() {
    return FALSE;
  }

  /**
   * {@inheritdoc}
   */
  public function hasMultipleValues(array $element) {
    if (!empty($element['#extra']['multiple']) || (!empty($element['#options']) && count($element['#options']) === 1)) {
      return TRUE;
    }
    return FALSE;
  }
```
`Array to string conversion in Drupal\webform\WebformSubmissionStorage->saveData() (line 1343 of /Applications/MAMP/htdocs/d9civicrm.local/web/modules/contrib/webform/src/WebformSubmissionStorage.php)`

**Comment by jitendrapurohit** [comment]
closing in favor of https://github.com/colemanw/webform_civicrm/pull/467

**Pull request closed without merging**